Formula/subfinder.rb

```diff
@@ -0,0 +1,56 @@
+require "language/go"
+class Subfinder < Formula
+  desc "Subdomain discovery tool"
+  homepage "https://github.com/subfinder"
+  head "https://github.com/subfinder/subfinder.git"
+
+  stable do
+    url "https://github.com/subfinder/subfinder/archive/1.2.tar.gz"
+    sha256 "2b8a09506466baa04c10404d38e0da6a2066a79e8b5c69a720d939804375ba2c"
+    go_resource "github.com/bogdanovich/dns_resolver" do
```

Comment:
Doesn't this tool use go modules?

Reply:
Would you have an example of another tool that does? I tried using this as a guide: https://github.com/Homebrew/homebrew-core/blob/master/Formula/mailhog.rb

Reply:
I think I figured it out, I added dep, is that okay?

Reply:
That's likely a pretty old formula. This is a better option to base on: https://github.com/Homebrew/homebrew-core/blob/master/Formula/teleport.rb. As long as dep doesn't download unversioned dependencies and has checksums for all of them that's fine too